cmd/cli/install_test.go

```diff
@@ -1407,57 +784,37 @@ var _ = Describe("Test envoy log level types", func() {
 	})
 })
 
-var _ = Describe("Test osm image pull policy cli option", func() {
-	It("Should correctly resolve the pull policy option to chart values", func() {
-		installCmd := &installCmd{
-			osmImagePullPolicy: "IfNotPresent",
-		}
-
-		vals, err := installCmd.resolveValues()
-		Expect(err).NotTo(HaveOccurred())
-
-		pullPolicy := vals["OpenServiceMesh"].(map[string]interface{})["image"].(map[string]interface{})["pullPolicy"]
-		Expect(pullPolicy).To(Equal("IfNotPresent"))
-	})
-
-	It("Should correctly resolve the pull policy option to chart values", func() {
-		installCmd := &installCmd{
-			osmImagePullPolicy: "Always",
-		}
-
-		vals, err := installCmd.resolveValues()
-		Expect(err).NotTo(HaveOccurred())
+func TestResolveValues(t *testing.T) {
+	assert := assert.New(t)
 
-		pullPolicy := vals["OpenServiceMesh"].(map[string]interface{})["image"].(map[string]interface{})["pullPolicy"]
-		Expect(pullPolicy).To(Equal("Always"))
-	})
-})
+	out := new(bytes.Buffer)
+	installCmd := getDefaultInstallCmd(out)
 
-var _ = Describe("Resolving values for service cert validity duration", func() {
-	It("Should correctly resolve the validity duration", func() {
-		installCmd := &installCmd{
-			serviceCertValidityDuration: "24h",
-		}
+	// Fill out fields which are empty by default
+	installCmd.containerRegistrySecret = testRegistrySecret
+	installCmd.vaultHost = testVaultHost
+	installCmd.vaultToken = testVaultToken
 
-		vals, err := installCmd.resolveValues()
-		Expect(err).NotTo(HaveOccurred())
+	expectedValues := getDefaultValues()
```

Comment:
I see both `getDefaultValues()` and `getDefaultInstallCmd()` are only used once in this test. Are they likely to be used in other tests in the future? If not, I would personally find it easier to read if the values were hardcoded right in the test vs. in separate functions.

Reply:
Yep, plan to use these in the future! For example, the first couple tests just test the default params. Then subsequent tests often just change on param. it'll help us easily read in a test exactly which fields are being tested since we seem to arbitrarily specify random fields in any given test.